### PR TITLE
Add a rpm-packaging box

### PR DESCRIPTION
**Pull request by mdellweg**
This box installs the dependencies to run add_gem_package.sh and clones
foreman-packaging in the users context.

### CODE REVIEW

File: playbooks/rpm_packaging.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          - redhat-rpm-config
+          - ruby-devel
+          - curl-devel
+          - gem
+          - python3-click
+          - python3-ruamel-yaml
+          - crudini
+      become: true
+
+    - name: Install gem-compare
```
Comment (ekohl) [inline playbooks/rpm_packaging.yml]: I use https://copr.fedorainfracloud.org/coprs/ekohl/gem-compare/ and saves the need for having `gem` and all `-devel` packages.

File: playbooks/rpm_packaging.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+- hosts: all
+  tasks:
+    - name: Install dependencies for packageing tooling
```
Comment (ekohl) [inline playbooks/rpm_packaging.yml]:
```suggestion
    - name: Install dependencies for packaging tooling
```

File: playbooks/rpm_packaging.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+- hosts: all
+  tasks:
+    - name: Enable Fedora Copr for gem-compair
```
Comment (ehelms) [inline playbooks/rpm_packaging.yml]: s/gem-compair/gem-compare

File: playbooks/rpm_packaging.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    - name: Install dependencies for packaging tooling
+      yum:
+        name: "{{ package_list }}"
+        state: present
+      vars:
+        package_list:
+          - git-core
+          - git-annex
+          - rubygem-nokogiri
+          - rubygem-gem-compare
```
Comment (ekohl) [inline playbooks/rpm_packaging.yml]: Looks like I locally depend on an ancient package still installed from Fedora 24 and COPR doesn't have repoclosure so it currently fails.

Comment (mdellweg) [reply playbooks/rpm_packaging.yml]: Yes. The question is, should we update the rpm, or drop the top commit from this PR?

Comment (ekohl) [reply playbooks/rpm_packaging.yml]: I'd also be ok changing the bumping script to not hard depend on gem-compare. Its primary benefit to me has been showing differences in the requirements. When we automatically update the dependencies, the biggest benefit is seeing if files were added/removed. The build will fail in most cases.

**Comment by ehelms** [comment]
What is the resolution here?

**Comment by mdellweg** [comment]
I can still remove the last commit. It will compile gem-compare then.
If there is a benefit, i think, we should keep the gem-compare dependency.

**Pull request merged**